src/components/Header.tsx

```diff
@@ -22,12 +22,14 @@ const Header = (props: BoxProps) => {
         <Flex justify="space-between" alignItems="center">
           <Flex alignItems="center">
             <Box h={LOGO_SIZES} w={LOGO_SIZES} mr={2}>
-              <Image
-                priority
-                src={mascotIcon}
-                alt="Octoclairvoyant reading a crystal ball"
-                placeholder="blur"
-              />
+              <RouteLink href="/">
```

Comment:
The `RouteLink` component has a really weird outline... The outline does not cover the full box of the image now. You can see this by refreshing the page and using <kbd>Tab</kbd> to tab into the image.

Reply:
Let me check this.

Reply:
I don't see any outline in this component, it's not focused on tab pressed 😕

Reply:
![bad_focus_outline](https://user-images.githubusercontent.com/34918129/126034264-1ee7398f-b5f6-44d2-a19e-3aab8c0634ae.png)

This picture shows the bad focus outline on the h1 image in Firefox 90.

Also, we have a problem with the background, see the white bar when the page is on fullscreen mode. I would have expected the light gray color to extend all the way to the footer.

Reply:
If you put the Box outside the RouterLink it gets fixed? I've seen it's semantically ok to have it like that, so if it works that way let's revert it.

About the background color, I'd say that's out of the scope of this PR, but definitely has to be fixed.

Reply:
> background color, I'd say that's out of the scope of this PR, but definitely has to be fixed.

It's out of scope, but wanted to mention it anyways. 😄 

I'll go make an issue to track this bug. 😉

Reply:
> If you put the Box outside the RouterLink it gets fixed? I've seen it's semantically ok to have it like that, so if it works that way let's revert it.

I'll go try, see if it fixes the bad focus outline. 😄

Reply:
![fix-bad-outline-commit-6dcb39a](https://user-images.githubusercontent.com/34918129/126034852-90c27e41-3f3c-4da4-98e2-8d71c849d6ed.png)

This is what we have now, with the new commit. The focus outline behaves a lot better now. 😄